src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -131,7 +131,7 @@ public string RebootAndroidDevice()
     public void EnableWifi(bool enable) => RunAdbCommand("shell", "svc", "wifi", enable ? "enable" : "disable")
         .ThrowIfFailed($"Failed to {(enable ? "enable" : "disable")} WiFi on the device");
 
-    public void DumpAdbLog(string outputFilePath, string filterSpec = "")
+    public bool DumpAdbLog(string outputFilePath, string filterSpec = "")

```

Comment:
nit: Maybe we could name this method TryDumpAdbLog, since it has a similar behaviour to the other Try.. methods

Reply:
Good point, added